tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAssetDeletionTaskStepTest.scala

```diff
@@ -0,0 +1,58 @@
+package com.linagora.tmail.james.jmap.publicAsset
+
+import java.net.URI
+
+import com.linagora.tmail.james.jmap.publicAsset.PublicAssetRepositoryContract.{CREATION_REQUEST, USERNAME}
+import org.apache.james.blob.api.{BucketName, HashBlobId}
+import org.apache.james.blob.memory.MemoryBlobStoreDAO
+import org.apache.james.core.Username
+import org.apache.james.server.blob.deduplication.DeDuplicationBlobStore
+import org.assertj.core.api.Assertions.{assertThat, assertThatCode}
+import org.junit.jupiter.api.{BeforeEach, Test}
+import reactor.core.scala.publisher.{SFlux, SMono}
+
+import scala.jdk.CollectionConverters._
+
+class PublicAssetDeletionTaskStepTest {
+  val USERNAME_2: Username = Username.of("username2")
+  val PUBLIC_ASSET_URI_PREFIX = new URI("http://localhost:8080")
+  val BLOB_ID_FACTORY = new HashBlobId.Factory()
+  val BLOB_STORE = new DeDuplicationBlobStore(new MemoryBlobStoreDAO, BucketName.DEFAULT, BLOB_ID_FACTORY)
```

Comment:
Can't those two be inlined in `beforeEach` as they are not eccessed by test cases?